receiver/gitproviderreceiver/documentation.md

```diff
@@ -34,38 +34,24 @@ Number of repositories in an organization
 | ---- | ----------- | ---------- |
 | {repository} | Gauge | Int |
 
-### git.repository.pull_request.approved.time
+### git.repository.pull_request.count
 
-The amount of time it took a pull request to go from open to approved
+The number of pull requests in a repository, categorized by their state (either open or merged)
 
-| Unit | Metric Type | Value Type |
-| ---- | ----------- | ---------- |
-| s | Gauge | Int |
+| Unit | Metric Type | Value Type | Aggregation Temporality | Monotonic |
+| ---- | ----------- | ---------- | ----------------------- | --------- |
+| {pull_request} | Sum | Int | Cumulative | true |
```

Comment:
You've set the type of the metric `git.repository.pull_request.count` to a monotonic `Sum`, but since the pull request count does not count closed pull requests, it can go down. Should it be a `Gauge`?

Reply:
You're correct. We're going to make these changes locally, and once tested will push up the changes. Thanks!